[2023-11-10T08:29:46.616+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: customers.save_customers_to_excel manual__2023-11-10T08:29:38.110521+00:00 [queued]>
[2023-11-10T08:29:46.639+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: customers.save_customers_to_excel manual__2023-11-10T08:29:38.110521+00:00 [queued]>
[2023-11-10T08:29:46.640+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-11-10T08:29:46.641+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-11-10T08:29:46.642+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-11-10T08:29:46.670+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): save_customers_to_excel> on 2023-11-10 08:29:38.110521+00:00
[2023-11-10T08:29:46.678+0000] {standard_task_runner.py:55} INFO - Started process 231 to run task
[2023-11-10T08:29:46.681+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'customers', 'save_customers_to_excel', 'manual__2023-11-10T08:29:38.110521+00:00', '--job-id', '391', '--raw', '--subdir', 'DAGS_FOLDER/customer_1.py', '--cfg-path', '/tmp/tmpkjsafpqz']
[2023-11-10T08:29:46.685+0000] {standard_task_runner.py:83} INFO - Job 391: Subtask save_customers_to_excel
[2023-11-10T08:29:46.771+0000] {task_command.py:389} INFO - Running <TaskInstance: customers.save_customers_to_excel manual__2023-11-10T08:29:38.110521+00:00 [running]> on host f75f09223c8b
[2023-11-10T08:29:46.871+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=customers
AIRFLOW_CTX_TASK_ID=save_customers_to_excel
AIRFLOW_CTX_EXECUTION_DATE=2023-11-10T08:29:38.110521+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-10T08:29:38.110521+00:00
[2023-11-10T08:29:47.068+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-11-10T08:29:47.157+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-11-10T08:29:47.230+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-11-10T08:29:47.243+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=customers, task_id=save_customers_to_excel, execution_date=20231110T082938, start_date=20231110T082946, end_date=20231110T082947
[2023-11-10T08:29:47.310+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-11-10T08:29:47.331+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
